client/src/components/Group/GroupCard.js

```diff
@@ -0,0 +1,83 @@
+import { Grid, Typography, Card, CardContent, CardMedia, CardActions, Button } from '@material-ui/core';
+import PeopleAltOutlinedIcon from '@material-ui/icons/PeopleAltOutlined';
+import { makeStyles } from '@material-ui/core/styles';
+
+
+const useStyles = makeStyles((theme) => ({
+
+    cardGrid: {
+      paddingTop: theme.spacing(8),
+      paddingBottom: theme.spacing(8),
+    },
+    card: {
+      height: '100%',
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    cardMedia: {
+      paddingTop: '60%',
+    },
+    cardContent: {
+      flexGrow: 1,
+    },
+    button: {
+        color: '#FFF',
+        background: theme.palette.primary.gradient,
+        marginTop: theme.spacing(1),
+        height: '3rem',
+        width: "100%",
+
```

Comment:
I believe the MUI buttons are capitalized by default and it's recommended by them. The Sketch file shows them without that styling and I would say all caps is better. But to change that default you can add a style:

textTransform: 'none',

to the button.